src/transport/tcp/SendSocket.cpp

```diff
@@ -22,6 +23,8 @@ void SendSocket::dial()
     }
 
     int connFd = sock.get();
+    noDelay(connFd);
```

Comment:
Looks perfect, one small thing. Could you add a `SendSocket::setSocketOptions(int connFd)` method where we set all the corresponding options (in this case `noDelay` and `quickAck`)?

I am doing the same for RecvSockets that spin: https://github.com/faasm/faabric/pull/408/files#diff-37584dd06015859ef6b67a6eea37c0b3671f76c11f010614ff9f80fb70375854

Reply:
PS: and also consider prefixing `set` as we do for other methods. So `noDelay` -> `setNoDelay` and `quickAck` -> `setQuickAck`.